examples/basic/src/actions/index.js

```diff
@@ -3,11 +3,10 @@ import * as Rx from 'rxjs';
 export const FETCH_USER_PENDING = 'FETCH_USER_PENDING';
 export const FETCH_USER_FULFILLED = 'FETCH_USER_FULFILLED';
 export const FETCH_USER_ABORTED = 'FETCH_USER_ABORTED';
+const BASE_URL = 'http://localhost:3000/items/';
 
 export const fetchUser = () => (
-  (actions, store) => Rx.Observable.of({ id: 1, name: 'Bilbo Baggins', timestamp: new Date() })
-    // Delaying to emulate an async request, like Rx.Observable.ajax('/api/path')
-    .delay(1000)
+  (actions, store) => Rx.Observable.ajax(BASE_URL)
```

Comment:
@jayphelps  I think that covers it.  Thanks!

On Tue, Jun 7, 2016 at 1:12 PM, Jay Phelps <notifications@github.com> wrote:

> In examples/basic/src/actions/index.js
> <https://github.com/redux-observable/redux-observable/pull/43#discussion_r66112509>
> :
>
> >
> >  export const fetchUser = () => (
> > -  (actions, store) => Rx.Observable.of({ id: 1, name: 'Bilbo Baggins', timestamp: new Date() })
> > -    // Delaying to emulate an async request, like Rx.Observable.ajax('/api/path')
> > -    .delay(1000)
> > +  (actions, store) => Rx.Observable.ajax(BASE_URL)
>
> @cmelion <https://github.com/cmelion> cool. possible for you to make
> those changes? 💃
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/redux-observable/redux-observable/pull/43/files/8f55d936d43882bd2008e2f90b71fd09a602b26e#r66112509>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAt8TyKAOqaEK_2dj0phrpekC7BhYOA_ks5qJaaDgaJpZM4IwCId>
> .
>